Test failures on windows with graphviz missing .bat files.

Today I started seeing failures in the tests like:

```python-traceback
cmd = ['dot.bat', '-Kdot', '-Tpng']
input = b'digraph {\n\tgraph [rankdir=BT]\n\t4770961679805687079 [label="" shape=box]\n\t-2718215776272399448 [label="" shape=...79 -> 1289659767108614192\n\t-1109490218808452559 [label="" shape=box]\n\t-2595565211807651805 [label="" shape=box]\n}'
capture_output = True, check = True, encoding = None, quiet = False
kwargs = {'stderr': -1, 'stdin': -1, 'stdout': -1}

    def run(cmd, input=None, capture_output=False, check=False, encoding=None,
            quiet=False, **kwargs):
        """Run the command described by cmd and return its (stdout, stderr) tuple."""
        log.debug('run %r', cmd)
    
        if input is not None:
            kwargs['stdin'] = subprocess.PIPE
            if encoding is not None:
                input = input.encode(encoding)
    
        if capture_output:
            kwargs['stdout'] = kwargs['stderr'] = subprocess.PIPE
    
        try:
            proc = subprocess.Popen(cmd, startupinfo=get_startupinfo(), **kwargs)
        except OSError as e:
            if e.errno == errno.ENOENT:
>               raise ExecutableNotFound(cmd)
E               graphviz.backend.ExecutableNotFound: failed to execute ['dot.bat', '-Kdot', '-Tpng'], make sure the Graphviz executables are on your systems' PATH

C:\Miniconda3\envs\test-environment\lib\site-packages\graphviz\backend.py:169: ExecutableNotFound
```

for example: https://github.com/dask/dask/pull/7230/checks?check_run_id=1911913930

I suspsect that something is wrong with the conda-forge package: graphviz 2.46.1 h93c640b_0 which was released today. 

I am going to pin graphviz to an older version in the tests

--- a/continuous_integration/environment-3.6.yaml
+++ b/continuous_integration/environment-3.6.yaml
@@ -37,7 +37,7 @@ dependencies:
   - crick
   - cytoolz
   - distributed
-  - graphviz=2.38.0  # pinned to avoid https://github.com/dask/dask/issues/7231
+  - graphviz
   - ipython
   - lz4
   - numba

--- a/continuous_integration/environment-3.7.yaml
+++ b/continuous_integration/environment-3.7.yaml
@@ -36,7 +36,7 @@ dependencies:
   - crick
   - cytoolz
   - distributed
-  - graphviz=2.38.0  # pinned to avoid https://github.com/dask/dask/issues/7231
+  - graphviz
   - ipython
   - lz4
   - numba

--- a/continuous_integration/environment-3.8-dev.yaml
+++ b/continuous_integration/environment-3.8-dev.yaml
@@ -36,7 +36,7 @@ dependencies:
   - crick
   - cytoolz
   - distributed
-  - graphviz=2.38.0  # pinned to avoid https://github.com/dask/dask/issues/7231
+  - graphviz
   - ipython
   - lz4
   - numba>=0.49

--- a/continuous_integration/environment-3.8.yaml
+++ b/continuous_integration/environment-3.8.yaml
@@ -36,7 +36,7 @@ dependencies:
   - crick
   - cytoolz
   - distributed
-  - graphviz=2.38.0  # pinned to avoid https://github.com/dask/dask/issues/7231
+  - graphviz
   - ipython
   - lz4
   - numba

